lib/index.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Do we need to keep all this boilerplate in all files? If we're providing a built version using Browserify, can't we just add a banner during the build step?